demos/showcase/src/app/app.component.html

```diff
@@ -1,7 +1,96 @@
+<div style="display: flex; padding: 16px; justify-content: space-between">
```

Comment:
can we do flex-start instead of space-between to match react

Reply:
Sure thing, I've updated it to match.